data-integration/persist/model.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/persist as _;
+
+type Employee record {|
+    readonly int id;
+    string name;
+    int age;
+    string phone;
+    string email;
+    string department;
+    Task[] tasks;
+|};
+
+type Task record {|
+    readonly int taskId;
+    string description;
+    TaskStatus status;
+    Employee employee;
```

Comment:
Shouldn't/can't this be just the employee ID?

Reply:
If we remove this, we won't have a relationship between the Employee and the Task.

Reply:
Assumed it could work with the Task[] relationship + id. Let's leave as is then.